hpat/hiframes/dataframe_pass.py

```diff
@@ -1218,11 +1233,11 @@ def _run_call_col_reduce(self, assign, lhs, rhs, func_name):
         for d in data_args:
             func_text += "  {} = hpat.hiframes.api.init_series({})\n".format(d + '_S', d)
             func_text += "  {} = {}.{}()\n".format(d + '_O', d + '_S', func_name)
-        func_text += "  data = np.array(({},))\n".format(
+            func_text += "  data = np.array(({},))\n".format(
             ", ".join(d + '_O' for d in data_args))
-        func_text += "  index = hpat.str_arr_ext.StringArray(({},))\n".format(
+            func_text += "  index = hpat.str_arr_ext.StringArray(({},))\n".format(
```

Comment:
Why you moved these lines under `for`? Are you sure that previous algorithm was wrong?

Reply:
No, I overlooked it in the difference before push. Reverted this change.